pkg/cmd/pipeline/start.go

```diff
@@ -33,7 +33,7 @@ import (
 
 var (
 	errNoPipeline      = errors.New("missing pipeline name")
-	errInvalidPipeline = errors.New("invalid pipeline name")
+	errInvalidPipeline = "invalid pipeline name %s in namespace %s"
```

Comment:
Would this read better as `pipeline name %s does not exist in namespace %s`?

Reply:
Fixed